pandas/core/generic.py

```diff
@@ -981,11 +981,23 @@ def rename(self, *args, **kwargs):
         level : int or level name, default None
             In case of a MultiIndex, only rename labels in the specified
             level.
+        errors : {'ignore', 'raise'}, default 'ignore'
```

Comment:
Not something that you should solve in this PR but we shouldn't need to duplicate docstrings here. Would be nice to use the Substitution decorator as required for this if you want to do a follow up PR

Reply:
I'll do a followup PR on that! Would you mind giving me an example?

Reply:
Sure here's one:

https://github.com/pandas-dev/pandas/blob/221be3b4adde0f45927803b1c593b56d4678faeb/pandas/core/frame.py#L665